python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -388,21 +396,34 @@ def generate_image(self, science, template, difference, center, scale,
         """
         if (source is None) ^ (flags is None):
             raise RuntimeError("Must pass both `source` and `flags` together.")
+        numpy_cutouts = {}
         if not self.config.use_footprint:
             sizes = self.config.sizes
             cutout_science, cutout_template, cutout_difference = [], [], []
             for i, s in enumerate(sizes):
                 extent = lsst.geom.Extent2I(s, s)
-                cutout_science.append(science.getCutout(center, extent))
-                cutout_template.append(template.getCutout(center, extent))
-                cutout_difference.append(difference.getCutout(center, extent))
+                science_cutout = science.getCutout(center, extent)
+                template_cutout = template.getCutout(center, extent)
+                difference_cutout = difference.getCutout(center, extent)
+                if save_raw_img:
+                    numpy_cutouts[f"sci_{s}"] = science_cutout.image.array
```

Comment:
Doesn't this then put the cutout size in the filename?   I'm not sure we want that--let's ask on #ml-reliability

Reply:
With the above code, the cutouts will be saved as two distinct file- 12345678_32.npy and 12345678_64.npy- in case someone passes [32, 64] as cutout sizes.

Reply:
I see.  Okay, I think that is a reasonable approach, then.